pom.xml

```diff
@@ -31,6 +31,9 @@
         <air.check.skip-modernizer>true</air.check.skip-modernizer>
         <air.check.skip-spotbugs>true</air.check.skip-spotbugs>
         <air.check.skip-pmd>true</air.check.skip-pmd>
+
+        <project.build.targetJdk>11</project.build.targetJdk>
+        <air.modernizer.java-version>8</air.modernizer.java-version>
```

Comment:
why 8 and not 11?

Reply:
Updating code to newer APIs is a separate task.